WORKSPACE

```diff
@@ -3,9 +3,9 @@ workspace(name = "angular_material")
 # Add nodejs rules
 http_archive(
   name = "build_bazel_rules_nodejs",
-  url = "https://github.com/bazelbuild/rules_nodejs/archive/0.6.0.zip",
-  strip_prefix = "rules_nodejs-0.6.0",
-  sha256 = "e8a2bb5ca51fbafb244bc507bcebcae33a63d969f47413b319a8dcce032845bf",
+  url = "https://github.com/bazelbuild/rules_nodejs/archive/99166f8eb7fc628ca561acf9f9a51a1c26edadad.zip",
```

Comment:
do you need this labelled? 0.7.0 was from Friday and should have what you need

Reply:
It doesn't really need to be, I can update it next time around